exe/next.sh

```diff
@@ -11,6 +11,7 @@ STRING
   mv Gemfile.tmp Gemfile
 
   ln -s Gemfile Gemfile.next
+  echo "Created a symlink. Added helper method to Gemfile."
```

Comment:
This is much better than what we had before! 😄 I think we should go even further:

```suggestion
  echo <<-MESSAGE
Your Gemfile has been modified to support dual-booting!
There's just one more step: modify your Gemfile to use a newer version of Rails using the \`next?\` helper method. 

For example, here's how to go from 5.2.3 to 6.0:

if next?
  gem "rails", "6.0.0"
else
  gem "rails", "5.2.3"
end
MESSAGE
```